pygments/lexers/qlik.py

```diff
@@ -0,0 +1,142 @@
+"""
+    pygments.lexers.qlik
+    ~~~~~~~~~~~~~~~~~~~~~
+
+    Lexer for the qlik scripting language
+
+    :copyright: Copyright 2006-2021 by the Pygments team, see AUTHORS.
+    :license: BSD, see LICENSE for details.
+"""
+
+import re
+
+from pygments.lexer import RegexLexer, include, bygroups, words
+from pygments.token import *
+from pygments.lexers._qlik_builtins import (
+    OPERATORS_LIST,
+    STATEMENT_LIST,
+    SCRIPT_FUNCTIONS,
+    CONSTANT_LIST,
+)
+
+__all__ = ["QlikLexer"]
+
+field_name_options = r"(\w|\[[\w\s]\]|\"[\w\s]\")"
+
+
+class QlikLexer(RegexLexer):
+    """
+    Lexer for qlik code, including .qvs files
+    """
+
+    name = "Qlik"
+    aliases = ["qlik", "qlikview", "qliksense", "qlikscript"]
+    filenames = ["*.qvs", "*.qvw"]
+
+    flags = re.IGNORECASE
+
+    tokens = {
+        # Handle multi-line comments
+        "comment": [
+            (r"\*/", Comment.Multiline, "#pop"),
+            (r"[^\*]+", Comment.Multiline),
+        ],
+        # Handle numbers
+        "numerics": [
+            (r"\b\d+\.\d+(e\d+)?[fd]?\b", Number.Float),
+            (r"\b\d+\b", Number.Integer),
+        ],
+        # Handle variable names in things
+        "interp": [
+            (
+                r"(\$\()(\w+)(\))",
+                bygroups(String.Interpol, Name.Variable, String.Interpol),
+            ),
+        ],
+        # Handle strings
+        "string": [
+            (
+                r"'",
+                String,
+                "#pop",
+            ),
+            include("interp"),
+            (r"[^'(\$\()]+", String),
+        ],
+        #
+        "assignment": [
+            (r";", Punctuation, "#pop"),
+            include("root"),
+        ],
+        "field_name_quote": [
+            (
+                r"(\")",
+                Keyword,
+                "#pop",
+            ),
+            include("interp"),
+            (r"[^\"(\$\()]+", Keyword),
```

Comment:
Same remark as above.

Also, do you really want this as `Keyword`? That sounds strange for a file name (not sure what a 'field name' is in Qlik). Maybe `String.Symbol` would do?